packages/block-editor/src/components/inserter/style.scss

```diff
@@ -16,6 +16,29 @@ $block-inserter-tabs-height: 44px;
 	}
 }
 
+.block-editor-inserter__popover.is-quick {
+	.components-popover__content {
+		border: none;
+
+		.block-editor-inserter__quick-inserter > * {
```

Comment:
The use of wildcards and first/last-child selectors here feels a little hard to parse. I think I understand what it does, and that this is all fine, but are there some explicit CSS classes we could've used instead?

Reply:
@jasmussen  Thank you for the review! I've thought about using explicit classes, but we can’t predict which type of items or how many will be visible. This seems the most future proof way. Because those components can be visible or hidden, and it’s possible someone may add in a new div later?

Reply:
I figured it might be something like that. But we can try this for now and see if any problems pop up, it seems to work in my tests. I'll restart the e2e tests and see if we can get them to pass, then we can land this. Thanks again.